### PR TITLE
Minor changes to the MaterialDialog

### PR DESCRIPTION
**Pull request by tomriddle25**
## Things to see 
-  Added InteractionType.kt of WEBRTC to the url launcher dialog.

### CODE REVIEW

File: app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt [hunk 1]
```diff
@@ -2,5 +2,6 @@ package io.qtalk.qgamejsinterfacetester.core
 
 enum class InteractionType {
     WEB_SHARING,
-    IN_CALL
+    IN_CALL,
+    WEBRTC
```
Comment (shahsurajk) [inline app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt:6]: rename to: 
```suggestion
    WEB_RTC
```

File: app/src/main/res/values/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="interactionTypes">
+        <item>"Web Share"</item>
+        <item>"In Call"</item>
+        <item>"WEBRTC"</item>
```
Comment (shahsurajk) [inline app/src/main/res/values/arrays.xml]:
```suggestion
        <item>"WebRTC"</item>
```

File: app/src/main/java/io/qtalk/qgamejsinterfacetester/MainActivity.kt [hunk 3]
```diff
@@ -72,19 +77,24 @@ class MainActivity : AppCompatActivity() {
             .negativeButton(text = "Test URL") {
                 openTestUrl()
             }
-            .checkBoxPrompt(text = "Make a preview call", onToggle = null)
-            .input(prefill = getUrlFromPrefOrIntent()) { materialDialog, charSequence ->
-                val urlToOpen = charSequence.toString()
+            .listItemsSingleChoice(R.array.interactionTypes, initialSelection = 1, waitForPositiveButton = false){ dialog: MaterialDialog, index: Int, text: String ->
```
Comment (shahsurajk) [inline app/src/main/java/io/qtalk/qgamejsinterfacetester/MainActivity.kt]: [optional] Missing formatting rules:
```suggestion
            .listItemsSingleChoice(R.array.interactionTypes, initialSelection = 1, waitForPositiveButton = false) { dialog: MaterialDialog, index: Int, text: String ->
```

**Review by shahsurajk — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt (hunk 1 above)
Comment (tomriddle25) [reply app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt:6]: @shahsurajk I think I had a discussion on this with @mishraanuruddh  and We went with WEBRTC as a single word. We've kept it that way in the QTalk client too.

File: app/src/main/java/io/qtalk/qgamejsinterfacetester/MainActivity.kt (hunk 3 above)
Comment (tomriddle25) [reply app/src/main/java/io/qtalk/qgamejsinterfacetester/MainActivity.kt]: 😅 I'll make the other small changes.

File: app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt (hunk 1 above)
Comment (shahsurajk) [reply app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt:6]: https://quiph.slack.com/archives/C743GE5B5/p1601459946000600 `WONTFIX`

**Review by shahsurajk — APPROVED** [review]

**Pull request merged**